src/main/java/me/totalfreedom/totalfreedommod/httpd/module/Module_bans.java

```diff
@@ -0,0 +1,51 @@
+package me.totalfreedom.totalfreedommod.httpd.module;
+
+import me.totalfreedom.totalfreedommod.TotalFreedomMod;
+import me.totalfreedom.totalfreedommod.admin.Admin;
+import me.totalfreedom.totalfreedommod.banning.BanManager;
+import me.totalfreedom.totalfreedommod.httpd.HTTPDaemon;
+import me.totalfreedom.totalfreedommod.httpd.NanoHTTPD;
+import me.totalfreedom.totalfreedommod.util.FLog;
+import java.io.File;
+
+public class Module_bans extends HTTPDModule
+{
+
+    public Module_bansTotalFreedomMod plugin, NanoHTTPD.HTTPSession session)
+    {
+        super(plugin, session);
+    }
+
+    @Override
+    public NanoHTTPD.Response getResponse()
+    {
+        File banFile = new File(plugin.getDataFolder(), BanManager.CONFIG_FILENAME);
+        if (banFile.exists())
+        {
+            final String remoteAddress = socket.getInetAddress().getHostAddress();
+            if (!isAuthorized(remoteAddress))
+            {
+                FLog.info("An unregistered IP (" + remoteAddress + ") has tried to access the ban list.");
+                return new NanoHTTPD.Response(NanoHTTPD.Response.Status.NOT_FOUND, NanoHTTPD.MIME_PLAINTEXT,
+                        "You may not view the banlist, Your IP, " + remoteAddress + ", is not registered to an admin on the server.");
```

Comment:
Do we really need to feed back the players IP? Don't see that being of any use...